script/c511009531.lua

```diff
@@ -10,7 +10,7 @@ function c511009531.initial_effect(c)
 	e1:SetCode(EVENT_SUMMON_SUCCESS)
 	e1:SetProperty(EFFECT_FLAG_CARD_TARGET+EFFECT_FLAG_DELAY)
 	e1:SetRange(LOCATION_MZONE)
-	e1:SetCountLimit(1)
+	e1:SetCountLimit(1,EFFECT_COUNT_CODE_SINGLE)

```

Comment:
Why was this changed to a Hard OPT?
http://yugioh.wikia.com/wiki/D/D/D_Gust_High_King_Executive_Alexander